engine/worker.go

```diff
@@ -50,3 +53,17 @@ func (e *Engine) initWorker() error {
 	e.worker = w
 	return nil
 }
+
+func initRuntime() (runtime.Runtime, error) {
+	runtimeType := conf.StringDefault("runtime.type", "docker")
+	switch runtimeType {
+	case "docker":
+		return docker.NewDockerRuntime()
+	case "shell":
+		return shell.NewShellRuntime(shell.Config{
+			CMD: conf.Strings("runtime.shell.cmd"),
+		}), nil
+	default:
```

Comment:
Would defaulting to Docker make more sense? Else this would be a breaking change.

Reply:
For sure. This is what the 2nd param on line 58 does.

Reply:
Ahhh, i see it :-)